examples/examples2d.jl

```diff
@@ -414,10 +414,14 @@ end
             linewidth=3)
 
         cm = colorlegend(
-            p1[end],           # access the plot of Scene p1
-            raw = true,        # without axes or grid
-            camera = campixel! # gives a concrete bounding box in pixels
+            p1[end],             # access the plot of Scene p1
+            raw = true,          # without axes or grid
+            camera = campixel!,  # gives a concrete bounding box in pixels
             # so that the `vbox` gives you the right size
+            width = (                     # make the colorlegend longer so it looks nicer
+                30,                       # the width
+                AbstractPlotting.reasonable_resolution()[end] # a reasonable height
```

Comment:
do you maybe want to use the actual window height here?

Reply:
I wasn't sure how to get that...tried to check the `Scene` attributes but couldn't find it there.

Reply:
Is is just assumed 960x540?

Reply:
size(scene) works, but gives you only the current value...
`lift(x-> widths(x)[2], pixelarea(scene))` will give you a signal that updates on resize!
Although, you don't have the final scene yet... Could create a scene in the beginning and then add to that - not sure if that would be worth the trouble :D... Would be nice to just have `with = (0.1rel, 1.0rel)` with window relative units...

Reply:
True that...I think for our purposes though a number would be sufficient :)

Reply:
(especially because this is supposed to be an educational example and adding Observables into the mix would cause more confusion)